page-objects/leftMenu.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+export class LeftMenu {
+    private readonly page: Page;
+    private readonly leftSideBar: Locator;
+
+    private menuElement: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.leftSideBar = this.page.locator("cu-simple-bar");
+    }
+
+    clickOnElement = async (elementName: string) => {
+        this.menuElement = this.leftSideBar.getByRole('treeitem', { name: elementName })
+        await this.menuElement.waitFor()
+        await this.menuElement.click()
+    };
```

Comment:
Do omówienia - czy powinniśmy w POM korzystać z arrow functions w ten sposób? Czy nie czyściej jest tradycyjnie definiować funkcje? Just noting, nie mam pojęcia czy ma to znaczenie.